packages/components/src/Placeholder/constants.ts

```diff
@@ -0,0 +1,25 @@
+import { Size } from './enums';
+
+export const IMAGE_WIDTH = {
+  [Size.Small]: '239px',
+  [Size.Medium]: '323px',
+  [Size.Large]: '458px',
+};
+
+export const IMAGE_HEIGHT = {
+  [Size.Small]: '119px',
+  [Size.Medium]: '161px',
+  [Size.Large]: '229px',
+};
+
+export const MAX_INNER_WIDTH = {
+  [Size.Small]: '384px',
+  [Size.Medium]: '400px',
+  [Size.Large]: '460px',
+};
+
+export const TITLE_HEADER_LEVEL = {
+  [Size.Small]: 'h5',
+  [Size.Medium]: 'h4',
+  [Size.Large]: 'h3',
+};
```

Comment:
```suggestion
} as const;
```
Тогда не нужно будет делать сдезт приведение
 <Title $size={size} variant={TITLE_HEADER_LEVEL[size] as 'h5'}>